ui/src/design-system/components/identification/identification-status/identification-status.stories.tsx

```diff
@@ -14,24 +14,26 @@ export default {
 
 export const Default: Meta = {
   args: {
+    alertThreshold: 0.6,
     isVerified: false,
     score: 0.7,
-    scoreThreshold: 0.6,
+    warningThreshold: 0.9,
```

Comment:
Ah yes! Should I also tweak the alert threshold? Suggestion:

[0.8, 1]: green
[0.6, 0.8]: orange
[0, 0.6]: red

I guess this could also be a deployment/project setting later on?